test/example/echo/echo.yaml

```diff
@@ -33,7 +33,10 @@ spec:
             "securityContext": {
               "privileged": true
             },
-            "command": ["/bin/bash", "-c", "sudo iptables -t nat -A PREROUTING -p tcp --dport 8080 -j REDIRECT --to-port 9999"]
+            "command": ["/bin/bash", "-c",
+              "sudo iptables -t nat -A PREROUTING -p tcp -j REDIRECT --to-port 5001",
+              "sudo iptables -t nat -A OUTPUT -p tcp -j REDIRECT --to-port 5002 -m owner \\! --uid-owner 1337"
+            ]
```

Comment:
Yeah, it has issues, but it works for a first iteration. @enricoschiattarella is looking for a better solution.